bundles/org.openhab.binding.energidataservice/README.md

```diff
@@ -605,6 +668,16 @@ console.log("Spot price two hours from now: " + price);
 
 :::
 
+::: tab JRuby
+
+```ruby
+two_hours_from_now = 2.hours.from_now.truncated_to(ChronoUnit::HOURS)
```

Comment:
Minor, but it's not two hours from now since truncated. Perhaps following the variable name of other language example will be simplest?
```suggestion
hour_start = 2.hours.from_now.truncated_to(ChronoUnit::HOURS)
```

Reply:
changed